modules/rostests/winetests/ddraw/math_workarounds.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#define _USE_MATH_DEFINES 1
+
+#include <math.h>
+
+#define exp2(x)         exp(M_LN2*(x))
```

Comment:
Have you checked how accurate that is? I have the feeling that `pow(2..0, x)` would yield better results.
Alternatively you could add the existing files to msvcrtex library.